packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -209,7 +209,8 @@ export default class Meetings extends WebexPlugin {
 
       if (data.locus && data.locus.replaces && data.locus.replaces.length > 0) {
         // Findes the first occurence of the meeting in the collection with same locus url
```

Comment:
Can you remove this comment since it is no longer appropriate?

Reply:
done